web/libs/datamanager/src/stores/Action.js

```diff
@@ -44,6 +44,7 @@ const ActionDialog = types.model("ActionDialog", {
   text: types.string,
   type: types.enumeration(["confirm", "prompt"]),
   form: types.maybeNull(types.array(ActionFormRow)),
+  modal_title: types.maybeNull(types.string),
```

Comment:
I think this is ideal here, as we already have distinguished it is an ActionDialog.
```suggestion
  title: types.maybeNull(types.string),
```